### PR TITLE
add time to jump, store LA city boundary as function

### PR DESCRIPTION
**Pull request by hunterowens**
This PR adds additional time for the JUMP etl, since the landing times on their trips endpoint has been fairly late. 

it also add a couple SQL scripts for referencing purposes.

### CODE REVIEW

File: dags/transportation/dockless/create_la_boundary.sql [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (ian-r-rose) [inline dags/transportation/dockless/create_la_boundary.sql]: Is this used anywhere at the moment, or do you intend to use it somewhere?

File: dags/transportation/dockless/v_trips.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
+CREATE MATERIALIZED VIEW v_trips AS
```
Comment (ian-r-rose) [inline dags/transportation/dockless/v_trips.sql]: Was this view missing before, or not checked in?

File: dags/transportation/dockless/v_trips.sql [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+SELECT provider_id,
+	   provider_name, 
+	   device_id,
+	   vehicle_id,
+	   vehicle_type,
+	   propulsion_type,
+	   trip_duration, 
+	   trip_distance, 
+	   trip_distance * 0.000621371192 as trip_distance_miles,
```
Comment (ian-r-rose) [inline dags/transportation/dockless/v_trips.sql]: Some people may recognize the conversion from meters to miles, but not most. Maybe a variable here would help for clarity.

File: dags/transportation/dockless/create_la_boundary.sql (hunk 1 above)
Comment (hunterowens) [reply dags/transportation/dockless/create_la_boundary.sql]: yeah, this function is used by folks when querying

File: dags/transportation/dockless/v_trips.sql (hunk 2 above)
Comment (hunterowens) [reply dags/transportation/dockless/v_trips.sql]: not checked in.

File: dags/transportation/dockless/v_trips.sql (hunk 3 above)
Comment (hunterowens) [reply dags/transportation/dockless/v_trips.sql]: hmmm good idea. I'll revise.

Comment (ian-r-rose) [reply dags/transportation/dockless/v_trips.sql]: Care to make this change? If not, we can move forwards and do it in a follow-up.

**Pull request merged**